internal/filtering/statefilter.go

```diff
@@ -4,9 +4,17 @@ import (
 	"github.com/snyk/snyk-iac-capture/internal/terraform"
 )
 
-var globalWhitelistedAttributes = []string{"id"}
+type StateFilterer struct {
```

Comment:
we can have an allowlist to pass to the filter function.
I'm especially talking about the filterer. I don't see why it would be better to have a filterer struct that we have to and really use one time only